packages/core/src/artifacts.ts

```diff
@@ -528,12 +527,11 @@ export const isContractDeploymentArtifact = (
 }
 
 /**
- * Make deployment artifacts for the most recent deployment in a project.
+ * Make deployment artifacts for the most recent deployment in a project. Mutates the input
+ * `artifacts` object.
```

Comment:
My comments for the `makeContractDeploymentArtifacts` function applies to this function too